nsqlookup/resolver.go

```diff
@@ -134,9 +133,78 @@ func (r *ConsulResolver) Resolve(ctx context.Context) (list []string, err error)
 		address = "http://" + address
 	}
 
-	if req, err = http.NewRequest("GET", address+"/v1/catalog/service/"+service, nil); err != nil {
+	// get list of check results for service
```

Comment:
This comment seems unnecessary, the code is pretty clear already, no need to be redundant.